merkle/compact/tree.go

```diff
@@ -138,11 +138,9 @@ func (t *Tree) String() string {
 	return buf.String()
 }
 
-// calculateRoot computes the current root hash. It calls visit function for
+// CalculateRoot computes the current root hash. It calls visit function for
```

Comment:
This might be a place to drop the ephemeral nodes comment from the changelog.

Reply:
Done.